### DISCUSSION TITLE
Command prompt displays node when node is not active

### DISCUSSION THREAD

**Discussion by ciarantr** [starship/starship#3583] [Q&A] [answered]
Hi, I'm having an issue with node constantly displaying in the command prompt. Has anyone had this issue or knows how to fix it ?

Starship : version latest
Terminal : Iterm2 version: latest
Shell: Zsh
<img width="1090" alt="Screenshot 2022-02-08 at 14 05 10" src="https://user-images.githubusercontent.com/58338331/153002820-b3b385e1-59b1-482b-989f-3c373d388ddf.png">

**Comment by matchai** [comment, marked as answer]
This is likely due to your root directory meeting any of the following criteria, as detailed in the [node module docs](https://starship.rs/config/#node-js):

- The current directory contains a package.json file
- The current directory contains a .node-version file
- The current directory contains a .nvmrc file
- The current directory contains a node_modules directory
- The current directory contains a file with the .js, .mjs or .cjs extension
- The current directory contains a file with the .ts extension

**Reply by ciarantr** [reply]
Thanks a lot @matchai! I missed the package.json in my root directory.
